ui/app/accounts/new-account/create-form.js

```diff
@@ -5,18 +5,20 @@ const { connect } = require('react-redux')
 const actions = require('../../actions')
 
 class NewAccountCreateForm extends Component {
-  constructor (props) {
-    super(props)
-    const { numberOfExistingAccounts = 0 } = props
```

Comment:
I think this can still be done in the `constructor` so that the new account name placeholder needs to be calculated only once instead of on every re-render. We can just use a different property in the state like `newAccountNamePlaceholder`.